source/Node.Extensibility.Tests/TinyTypes/StringEqualityTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using NUnit.Framework;
+using Octopus.Server.Extensibility.TinyTypes;
+
+namespace Node.Extensibility.Tests.TinyTypes
+{
+    public class StringEqualityTests
```

Comment:
Should these be in here? Or should they be in `TinyTypes` itself?

Reply:
Good point. I did think about this :)

I'd expect that we'd pull these out into the TinyTypes library soon-ish but there are some design differences (mainly around implicit cast operators) that I'd like to discuss first. I didn't want to introduce yet another dependency in the chain that would be so far removed from where we're actually trying to use it, which is already one consumer upstream from here.